src/clientenvironment.cpp

```diff
@@ -94,119 +94,21 @@ void ClientEnvironment::step(float dtime)
 	/* Step time of day */
 	stepTimeOfDay(dtime);
 
+	// Don't allow overly huge dtime
+	if (dtime > 0.5)
+		dtime = 0.5;
+
 	// Get some settings
-	bool fly_allowed = m_client->checkLocalPrivilege("fly");
-	bool free_move = fly_allowed && g_settings->getBool("free_move");
+	PlayerSettings settings;
```

Comment:
same here

Reply:
As above, please see #7243 and the above answer.

Reply:
Fixed due to the merge of #7243